rpcserver/rpcserver.go

```diff
@@ -60,6 +63,32 @@ func (h *RPCServerHandler) GetBTCHeaderChainTip() (Block, error) {
 	return latestFinalizedBlock, nil
 }
 
+// returns if the spvProof is valid or not
+func (h *RPCServerHandler) VerifySPV(spvProof *btclightclient.SPVProof) (btclightclient.SPVStatus, error) {
+	log.Info().Msgf("Recieved spvProof %v", spvProof)
+
+	checkSPV := h.btcLC.VerifySPV(*spvProof)
+
+	if checkSPV == btclightclient.ValidSPVProof {
```

Comment:
I think just show some message like `SPV proof: ${status}`. Don't need if here. 
So complicated for logger.